src/ui/Form/Checkbox.stories.js

```diff
@@ -89,15 +89,15 @@ function CheckboxForm({
         ) : null}
         <Form.Field error={invalid}>
           <Checkbox label="Option 1" defaultChecked id="field1"></Checkbox>
-          <label htmlFor="field1">Helper text for option 1</label>
+          <p class="description">Helper text for option 1</p>
```

Comment:
@g-stamatis Hmm. I'm not sure about this change.

checkboxes, radio, text inputs should have the label with the `for` attribute present.

Reply:
@avoinea The semantic uses the label prop inside the Checkbox component (line 91) so we had accesibility warnings for adding another label for the helper text. I don't clearly remember but I think we talked in a daily meeting and decided to just put a description p.